sopel/coretasks.py

```diff
@@ -59,6 +59,12 @@ def auth_after_register(bot):
             account + ' ' + password
         ))
 
+    elif bot.config.core.auth_method == 'userserv':
+	account = bot.config.core.auth_username
+	password = bot.config.core.auth_password
+	bot.msg("USERSERV", "LOGIN {account} {password}".format(
+		account=account, password=password))
+
```

Comment:
This block adds a `TabError` because some of the lines use tabs instead of spaces for indentation.

Reply:
fixed